### ISSUE TITLE
Regression bug on excessive ARPA lookups in DNS?

### ISSUE THREAD

**Issue by thefunkygibbon** [pucherot/Pi.Alert#259]
This happened a year or so ago and it was deduced that it was a bug and was resolved, but i've come to check my DNS logs (pihole) and i'm seeing the same as last time

![image](https://github.com/user-attachments/assets/6eaa4305-50ac-4892-918e-31f0e728f03d)

**Comment by jokob-sk** [comment]
Please close this issue and open it at the correct github repo. Look at https://github.com/pucherot/Pi.Alert/issues/235 and https://github.com/pucherot/Pi.Alert/issues/237

**Comment by mrcsrplln** [comment]
@thefunkygibbon  I had the same issues and received help from the great guys at PiHole - https://discourse.pi-hole.net/t/sudden-increase-in-local-ptr-queries-from-localhost-pihole-itself/72110/3 

In my case it was indeed PiAlert (Docker Container jokobsk/pi.alert:latest) that started spamming PTR queries every other minute.

**Comment by thefunkygibbon** [comment]
thanks, maybe I'm missing something since I've only just awoken, but I don't see any "help" as such or any way of fixing the issue , other than "turning it off" which isn't a great solution if you use the software lol.

**Comment by mrcsrplln** [comment]
> thanks, maybe I'm missing something since I've only just awoken, but I don't see any "help" as such or any way of fixing the issue , other than "turning it off" which isn't a great solution if you use the software lol.


Indeed, I was just confirming the issue you experienced. I ended up removing Pi Alert Docker Container as I haven't been using it at all.

**Comment by thefunkygibbon** [comment]
ah fair enough, I was trying to use it as a replacement for "fing" but its not got anywhere the amount of features that fing does so i've barely used it either.  guess i'll just ditch it.  shame really. 

thanks for the confirmation though dude, and sorry if my above message came across as a bit abrupt.

**Comment by jokob-sk** [comment]
Hey @thefunkygibbon - can you close this issue as you switched to the NetAlertX fork?
